R/data_meredith.R

```diff
@@ -0,0 +1,353 @@
+#'   \item{pricepercent}{The unit price percentile compared to the rest of the set.}
+#'   \item{winpercent}{The overall win percentage according to 269,000 matchups.}
+#' }
+#' @source See \url {https://github.com/fivethirtyeight/data/tree/master/candy-power-ranking}
+#'  @examples
+#' # To convert data frame to tidy data (long) format, run:
+#' library(tidyverse)
+#' library(stringr)
+#' candy_rankings_tidy <- candy_rankings %>%
+#'   gather(characteristics, present, -c(competitorname, sugarpercent, pricepercent, winpercent)) %>%
+#'   mutate(present = as.logical(present)) %>%
+#'   arrange(competitorname)
+"candy_rankings"
+
+
+
+#' Chess Transfers
+#'
+#' The raw data behind the story
+#' "American Chess Is Great Again"
+#' \url{https://fivethirtyeight.com/features/american-chess-is-great-again/}.
+#'
+#' @format A data frame with 932 rows representing international player transfers and 5 variables:
+#' \describe{
+#'   \item{url}{The corresponding website on the World Chess Federation page which details the transfers of a given year.}
+#'   \item{id}{An numeric identifier for the chess player who transfered.}
+#'   \item{federation}{The current national federation of the chess player}
+#'   \item{former_fed}{The national federation from which the chress player has transfered.}
+#'   \item{transfer}{The date at which the transfer took place.}
+#' }
+#' @source World Chess Federation \url{https://ratings.fide.com/fedchange.phtml}
+"chess_transfers"
+
+
+
+#' Congress Generic Ballot Polls 
+#'
+#' The raw data behind the story
+#' "Are Democrats Winning The Race For Congress?"
+#' \url{https://projects.fivethirtyeight.com/congress-generic-ballot-polls/}.
+#'
+#' @format A data frame with 934 rows representing polls and 21 variables:
+#' \describe{
+#'   \item{subgroup}{No description provided.}
+#'   \item{modeldate}{No description provided.}
+#'   \item{startdate}{Start date of the poll.}
+#'   \item{enddate}{End date of the poll.}
+#'   \item{pollster}{The organization that conducted the poll (rather than the organization that paid for or sponsored it)}
+#'   \item{grade}{No description provided.}
+#'   \item{samplesize}{No description provided.}
+#'   \item{population}{No description provided.}
+#'   \item{weight}{No description provided.}
+#'   \item{influence}{No description provided.}
+#'   \item{dem}{No description provided.}
+#'   \item{rep}{No description provided.}
+#'   \item{adjusted_dem}{No description provided.}
+#'   \item{adjusted_rep}{No description provided.}
+#'   \item{multiversions}{No description provided.}
+#'   \item{tracking}{No description provided.}
+#'   \item{url}{No description provided.}
+#'   \item{poll_id}{No description provided.}
+#'   \item{question_id}{No description provided.}
+#'   \item{created_date}{No description provided.}
+#'   \item{timestamp}{No description provided.}
+#' }
+#' @source See \url{https://github.com/fivethirtyeight/data/blob/master/congress-generic-ballot/README.md}
```

Comment:
Since `generic_polllist` and `generic_topline` are two datasets that are related to each other (come from the same article), could you add a `@seealso` type link for both of them? For an example, see how the following three help files link to each other: `?college_all_ages`, `?college_grad_students`, and `?college_recent_grads`. The roxygen help file code are in: `R/data_albert.R`